test/components/upgrades.test.js

```diff
@@ -224,6 +224,14 @@ describe('Upgrades testing', function () {
                 unsafeAllow: ['delegatecall'],
             });
             await this.stakingParameters.deployed();
+            const ScannerRegistry_0_1_3 = await ethers.getContractFactory('ScannerRegistry_0_1_3');
+            // Router is deprecated, just set an address
```

Comment:
We have a MockRouter we should use!

Reply:
Not in ScannerRegistry_0_1_3 actually, the comment is just misleading. Removing.